src/RVRP_data.jl

```diff
@@ -3,10 +3,14 @@ struct Range
     soft_min::Float64 # must be greater or equal to the hard_opening; can be undefined
     soft_max::Float64 # must be greater or equal to the soft_opening; can be undefined
     hard_max::Float64 # must be greater or equal to the soft_closing
+    nominal_unit_price::Float64 # to measure the cost/reward per unit 
     shortage_extra_unit_price::Float64 # to measure the cost/reward of being below this range's soft_opening
     excess_extra_unit_price::Float64 # to measure the cost/reward of being above this range's soft_closing
+    constraint_violation_status::Int # 0 means that hard bounds are mandatory, -1 means that even hardbounds are soft, a strictly positive values mean that harbounds are semi-mandatory, i.e. that they can be violated only if there does not exist any feasible solutions without violation. Then, the value of the constraint_violation_status defined a hierarchy of violation abong such constraint; the higher the value the most priority is put on enforcing them
```

Comment:
The problem with this is that we can t model the case where there are no more vehicles to lease, there is no more person at home to pick there is no more capacity in delivery warehouse. It is not just a matter of cost. In real life there is a range for flexibility but there is also a limit beyond which it s not just a matter of price. This modeling allows infinite violation, i.e. for some type of ranges the cost function never reaches a vertical line (with infinite slope) ...